pages/common/you-get.md

```diff
@@ -0,0 +1,24 @@
+# you-get
+
+> Download media contents (videos, audios, images) from the Web.
+> More information: <https://you-get.org>.
+
+- See all media information from web:
+
+`you-get -i {{https://example.com/video?id=jNQXAC9IVRw}}`
```

Comment:
```suggestion
`you-get --info {{https://example.com/video?id=value}}`
```

I am not sure whether it's a good idea to demonstrate concrete ids because such urls maybe invalidated in the future. 🤔 I prefer write smth more general.